dz22_ex3.sql

```diff
@@ -0,0 +1 @@
+select books.title, fname, lname from books join (select purchase.id, purchase.book_id as books, users.first_name as fname, users.last_name as lname from purchase join users on purchase.user_id = users.id) on books.id = books order by fname, lname
```

Comment:
1. `user.id` is missing in the results of this query
2. try to create a query without nested query but using join 2 times: `select ... join ... join ... `